scripts/drag_drop_handler.lua

```diff
@@ -0,0 +1,37 @@
+-- this script gets two parameters in its global scope:
```

Comment:
Nitpicky, but this sentence should start with a capital letter

Reply:
not need for "but";  "nitpicky" and "right" are not opposites